buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -3,6 +3,43 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": true,
```

Comment:
Ok. In that case let's try the following sequence:

1. `build.sh`
2. `pkg/init-tools.sh` 
3. `pkg/Tools/scripts/docker/cleanup-docker.sh` after 

No changes to 1.  But for 2 & 3, _continueOnError_ and _alwaysRun_ will be true. 
Concerns?